launchable/commands/record/session.py

```diff
@@ -50,7 +48,18 @@ def session(build_name: str, save_session_file: bool, print_session: bool = True
     # TODO: check duplicate keys
     flavor_dict = {}
     for f in flavor:
-        flavor_dict[f[0]] = f[1]
+        if "=" not in f:
```

Comment:
I think we can support both `key=val` and `key val` with `nargs=-1` option and [custom type](https://click.palletsprojects.com/en/7.x/parameters/#implementing-custom-types). Don't you think we need to support them?

Reply:
Oh, really? thank you for your advice.

I'll read it and try to support both

Reply:
@ninjinkun I tried it but couldn't. sorry,
because the `nargs` option only provides click.argument not supports click.option.

I tried to use a [click.argument](https://click.palletsprojects.com/en/7.x/arguments/) for the flavor option, but the click.argument can't set option name and some problems to implement it.

so unfortunately, I decided to keep this change

Reply:
but I just found it https://stackoverflow.com/questions/48391777/nargs-equivalent-for-options-in-click
so I tried it